### ISSUE TITLE
Tech review needed for merged PR: 

### ISSUE THREAD

**Issue by yochannah** [open-life-science/open-life-science.github.io#795]
https://github.com/open-life-science/open-life-science.github.io/pull/788

**Comment by bebatut** [comment]
My comments on my 1st pass on the PR:

- Emmy should be removed from organizers and Taj added
- I am not sure to understand why concatenating all cohorts in https://github.com/open-life-science/open-life-science.github.io/blob/main/_includes/cohort-loop.html#L9
- Why OLS-1 in https://github.com/open-life-science/open-life-science.github.io/blob/main/openseeds/ols-8/index.md?plain=1#L15 ?

**Comment by NPDebs** [comment]
Thanks for the technical review, @bebatut!

1. > Emmy should be removed from organizers and Taj added

This has been done in #790. 🙏 

2. > ...concatenating all cohorts...

What I did was to use the `concat` filter to combine two arrays (`openseeds_cohorts` & `catalyst_cohort`s) into one.
 ```
{%- assign cohorts = openseeds_cohorts | concat: catalyst_cohorts -%}
```
The result is stored in a new variable `cohorts`, which is subsequently looped over to extract unique program data. While it is no longer applicable to the structure of ols-9/catalyst, it's useful to keep in mind for future programs (e.g nebula).

3. > Why OLS-1 in...

That's a bug which was also fixed in #790.

Did I explain these adequately?
